client/js/screens/SignUp/SignUp.js

```diff
@@ -1,9 +1,104 @@
-import React from "react";
-import { Text } from "react-native";
+import React, { Fragment } from "react";
+import {
+  Text,
+  ImageBackground,
+  View,
+  TextInput,
+  TouchableOpacity
+} from "react-native";
 import styles from "./styles";
+import { Form, Field } from "react-final-form";
 
-const Signup = () => {
-  return <Text>I am the Signup Screen</Text>;
+const onSubmit = async (values, createUser) => {
+  const res = await createUser({ variables: values });
+};
+
+const Signup = ({ data }) => {
+  return (
+    <Fragment>
+      <ImageBackground
+        source={require("../../assets/images/Signup.png")}
+        style={styles.backgroundImage}
+      >
+        <Text style={styles.register}>REGISTER</Text>
+        <Form
+          onSubmit={values => onSubmit(values, createUser)}
+          render={({ handleSubmit, pristine, invalid }) => (
+            <View>
```

Comment:
Use Fragment instead. Can't have View component by itself.

Reply:
Remove all the View components that are just wrappers and nothing else and use Fragment instead.

Reply:
done